src/py/reactpy/reactpy/backend/__init__.py

```diff
@@ -0,0 +1,23 @@
+import mimetypes
+from logging import getLogger
+
+_logger = getLogger(__name__)
+
+# Fix for missing mime types due to OS corruption/misconfiguration
+# Example: https://github.com/encode/starlette/issues/829
+if not mimetypes.inited:
+    mimetypes.init()
+MIME_TYPES = {
+    ".js": "application/javascript",
+    ".css": "text/css",
+    ".json": "application/json",
+}
+for extension, mime_type in MIME_TYPES.items():
```

Comment:
avoids exposing `MIME_TYPES` as something importable and, IMO, doesn't really harm readability.

```suggestion
for extension, mime_type in {
    ".js": "application/javascript",
    ".css": "text/css",
    ".json": "application/json",
}.items():
```